src/main/java/com/wwi21sebgroup5/cinema/controller/EventController.java

```diff
@@ -0,0 +1,143 @@
+package com.wwi21sebgroup5.cinema.controller;
+
+import com.wwi21sebgroup5.cinema.entities.Event;
+import com.wwi21sebgroup5.cinema.requestObjects.EventRequestObject;
+import com.wwi21sebgroup5.cinema.services.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(path = "/v1/event")
+public class EventController {
+
+    @Autowired
+    EventService eventService;
+
+    /**
+     *
+     * @param requestObject RequestObject that holds the necessary information to add an event, this includes a
+     *                      movie id, a cinema hall id, a start date and an end date
+     * @return Returns the newly added event embedded in a ResponseEntity
+     */
+    @PostMapping(path = "/add")
+    public ResponseEntity<Object> addEvent(@RequestBody EventRequestObject requestObject) {
+        Event newEvent;
+
+        try {
+            newEvent = eventService.addEvent(requestObject);
+        } catch (Exception ex) {
```

Comment:
addEvent, kann 3 spezifische Exceptions werfen, warum diese nicht einzeln abfangen und den genauen Fehler am Endpunkt ausgeben? So wie es in den anderen Controllern auch gemacht wurde

Reply:
hab ich vergessen, änder ich